### ISSUE TITLE
Broken gift dependency in gulp-gh-pages

### ISSUE THREAD

**Issue by avdeev** [shower/shower#318]
Discussion https://github.com/shinnn/gulp-gh-pages/pull/117

```
Mac-mini-Alexey:jsonapi-work-according-to-specification alexeyavdeev$ npm run publish

> shower@2.3.0 publish /Users/alexeyavdeev/projects/jsonapi-work-according-to-specification
> gulp publish

[18:45:49] Using gulpfile ~/projects/jsonapi-work-according-to-specification/gulpfile.js
[18:45:49] Starting 'publish'...
[18:45:49] Starting 'prepare'...
[18:45:50] Finished 'prepare' after 1.13 s
[18:45:50] Starting 'upload'...
/Users/alexeyavdeev/projects/jsonapi-work-according-to-specification/node_modules/gulp-gh-pages/node_modules/gift/lib/commit.js:145
      ref1 = /^.+? (.*) (\d+) .*$/.exec(line), m = ref1[0], actor = ref1[1], epoch = ref1[2];
                                                       ^

TypeError: Cannot read property '0' of null
    at Function.module.exports.Commit.actor (/Users/alexeyavdeev/projects/jsonapi-work-according-to-specification/node_modules/gulp-gh-pages/node_modules/gift/lib/commit.js:145:56)
    at Function.module.exports.Commit.parse_commits (/Users/alexeyavdeev/projects/jsonapi-work-according-to-specification/node_modules/gulp-gh-pages/node_modules/gift/lib/commit.js:111:21)
    at /Users/alexeyavdeev/projects/jsonapi-work-according-to-specification/node_modules/gulp-gh-pages/node_modules/gift/lib/commit.js:55:39
    at ChildProcess.exithandler (child_process.js:267:7)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Socket.stream.socket.on (internal/child_process.js:346:11)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
```

**Comment by pepelsbey** [comment]
Is there a better `gulp-gh-pages` alternative? Maybe we could replace it with the one that doesn’t fail. 

**Off topic:** Speaking of which… if you’re interested working on Shower I’ll be glad to add you to contributors team that will give you push access. I’d still expect you to file PR for bigger changes so we could discuss them, but for minor tweaks the freedom is yours. What do you think?

**Comment by avdeev** [comment]
Apparently the project **gulp-gh-pages** is abandoned.

We can switch to forks and wait. :)

1) `"gulp-gh-pages": "git@github.com:tekd/gulp-gh-pages.git#update-dependency"` (work for me)
2) https://www.npmjs.com/package/gulp-gh-pages-will

Or else I found this package - https://github.com/tschaub/gh-pages
It's more general and gulp agnostic. :) We can try to switch to it.

**Off topic**: @pepelsbey, yes. It would be a good experience. Thanks :)

**Comment by pepelsbey** [comment]
You’re contributor now :) Welcome!

If you have time to replace `gulp-gh-pages` with `gh-pages`, please go fot it and let me know if I can help you in any way.

**Comment by avdeev** [comment]
Awesome! :) I'll do it next week. :)

**Comment by avdeev** [comment]
Solved in #320
